src/client/index.js

```diff
@@ -81,6 +82,15 @@ export const checkNumber = (value, name = "input number")=>{
   }
 }
 
+const checkTransfers = (transfers) => {
+  for(let i=0,len=transfers.length;i<len;i++) {
```

Comment:
Why not use foreach here, It would be more elegant..

Reply:
right. foreach is better here